### DISCUSSION TITLE
latest image doesnt containe latest changes to RTL_433

### DISCUSSION THREAD

**Discussion by fgonza2** [hertzg/rtl_433_docker#41] [Q&A] [answered]
hi, i am trying to use your image, but i see that is not based on the latest rtl_433.    Noticed this as i am looking for protocol 192, the current image only supports up to 186.   I can see on the rtl_433 that it is now at 200 protocols. 

Is ther a way to get the latest?

**Comment by hertzg** [comment]
Hello, Which image tag are you refering to?

**Comment by fgonza2** [comment]
hertzg/rtl_433:latest

**Comment by hertzg** [comment, marked as answer]
**TL;DR**: You want to use the `hertzg/rtl_433:master` image.

It is working as intended, `latest` docker tag corresponds to the `latest` release (git tag) of `rtl_433` which is `21.05` at the time of writing this and contains only 186 protocols. 

I'm assuming you want to use the `master` branch builds, which are also provided here but under a different tag (`:master`). 

You can see all mappings between git revisions and builds including their naming conventions in the README.
https://github.com/hertzg/rtl_433_docker#docker-image-tags

**Reply by sgofferj** [reply]
I came here following the issue and I somewhat have the same issue. However, I tried hertzg/rtl_433:master and it seems to have even less decoders

```Registered 148 out of 178 device decoding protocols [ 1-4 8 11-12 15-17 19-21 23 25-26 29-36 38-60 63 67-71 73-100 102-105 108-116 119 121 124-128 130-149 151-161 163-168 170-175 177-178 ]```

The version it spits out is ```rtl_433 version 20.11-72-g914552b9 branch master at 202102121205 inputs file rtl_tcp RTL-SDR```

**Reply by hertzg** [reply]
This is now tracked in #47. PR #49 includes the fix for it, should be updated once it lands. 🚀

**Reply by sgofferj** [reply]
Awesome! Thanks!

**Comment by hertzg** [comment]
I'm going to mark the latest reply as answer as there's been no reply on it from last week. Feel free to comment if it's not resolved.

**Comment by josemariaaraujo** [comment]
Sorry, I just craeted an issue with the same problem, and found this discussion later.
`hertzg/rtl_433:master` hasn't been updated in the last 10 months. 
The solution I found is to use `hertzg/rtl_433:alpine-master` tag, that has the latest updates. It reports `rtl_433 version 21.05 (2021-05-09)` but I'm using rtl_tcp settings which were only recently supported.

**Reply by hertzg** [reply]
Thanks for reporting this, I think this feature was lost with the big refactoring #8 (which did not help much the code is still a mess 🙈 ) I've started migrating to ESM modules and included a quick (hacky) fix for this in the same pr. It should fix the issue.

**Comment by buzz-b** [comment]
Hi @hertzg, any chance we can get an updated version that uses at least rtl_version "nightly-21-gfe3df68d branch master at 202303301119". There have been a large number of additional streams added since the release 22.11 was launched. Unfortunately the I need is Resource 240 now, so it's not in your docker image. I got your docker image working well on my QNAP NAS, so that awesome, but it doesn't understand my weather device.

Please could you update the docker image accordingly, that would be great.
Thanks,
Andy...
